ui/selectors/selectors.js

```diff
@@ -927,6 +927,23 @@ export const getMemoizedTargetSubjectMetadata = createDeepEqualSelector(
   (interfaces) => interfaces,
 );
 
+/**
+ * Get a memoized version of the unapproved confirmations.
+ */
+export const getMemoizedUnapprovedConfirmations = createDeepEqualSelector(
+  getUnapprovedConfirmations,
+  (confirmations) => confirmations,
+);
+
+/**
+ * Get a memoized version of the unapproved templated confirmations.
+ */
+export const getMemoizedUnapprovedTemplatedConfirmations =
```

Comment:
If we know the data is complex enough to warrant deep equality checks, would it be better to update the existing selectors with deep equality rather than creating parallel selectors?

Is there a downside if it reduces the number of renders, maybe a tiny performance loss during comparison?

Reply:
It is not always possible to replace the previous selector entirely as not the entire app uses reselect. Therefore it often is needed to have a duplicate selector that is memoized and one that isn't.